src/main/java/io/jenkins/plugins/coverage/model/CoverageStatistics.java

```diff
@@ -0,0 +1,151 @@
+package io.jenkins.plugins.coverage.model;
+
+import java.io.Serializable;
+import java.util.List;
+import java.util.Map;
+import java.util.SortedMap;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+
+/**
+ * Class that represents a coverage statistic.
+ */
+public class CoverageStatistics {
+    private Map<String, Coverage> sortedCoverages;
```

Comment:
`sorted` is misleading, wouldn't it make more sense to name it `coveragesByMetric`?